site/content/en/docs/components/_index.md

```diff
@@ -13,61 +13,61 @@ Prow has a microservice architecture implemented as a collection of container im
-* `mkpj` ([doc](/docs/components/cli-tools/mkpj/), [code](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/mkpj)) creates `ProwJobs` using Prow configuration.
-* `mkpod` ([doc](/docs/components/cli-tools/mkpod/), [code](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/mkpod)) creates `Pods` from `ProwJobs`.
-* `peribolos` ([doc](/docs/components/cli-tools/peribolos/), [code](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos)) manages GitHub org, team and membership settings according to a config file. Used by [kubernetes/org](https://github.com/kubernetes/org)
-* `phaino` ([doc](/docs/components/cli-tools/phaino/), [code](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/phaino)) runs an approximation of a ProwJob on your local workstation
-* `phony` ([doc](/docs/components/cli-tools/phony/), [code](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/phony)) sends fake webhooks for testing hook and plugins.
+* `checkconfig` ([doc](/docs/components/cli-tools/checkconfig/), [code](https://github.com/kubernetes-sigs/prow/tree/main/prow/cmd/checkconfig)) loads and verifies the configuration, useful as a pre-submit.
+* `config-bootstrapper` ([doc](/docs/components/cli-tools/config-bootstrapper/), [code](https://github.com/kubernetes-sigs/prow/tree/main/prow/cmd/config-bootstrapper)) bootstraps a configuration that would be incrementally updated by the [`updateconfig` Prow plugin](/docs/components/plugins/updateconfig/)
+* `generic-autobumper` ([doc](/docs/components/cli-tools/generic-autobumper/), [code](https://github.com/kubernetes-sigs/prow/tree/main/prow/cmd/generic-autobumper)) automates image version upgrades (e.g. for a Prow deployment) by opening a PR with images changed to their latest version according to a config file.
+* `invitations-accepter` ([doc](/docs/components/cli-tools/invitations-accepter/), [code](https://github.com/kubernetes-sigs/prow/tree/main/prow/cmd/invitations-accepter)) approves all pending GitHub repository invitations
+* `mkpj` ([doc](/docs/components/cli-tools/mkpj/), [code](https://github.com/kubernetes-sigs/prow/tree/main/prow/cmd/mkpj)) creates `ProwJobs` using Prow configuration.
+* `mkpod` ([doc](/docs/components/cli-tools/mkpod/), [code](https://github.com/kubernetes-sigs/prow/tree/main/prow/cmd/mkpod)) creates `Pods` from `ProwJobs`.
+* `peribolos` ([doc](/docs/components/cli-tools/peribolos/), [code](https://github.com/kubernetes-sigs/prow/tree/main/prow/cmd/peribolos)) manages GitHub org, team and membership settings according to a config file. Used by [kubernetes/org](https://github.com/kubernetes/org)
+* `phaino` ([doc](/docs/components/cli-tools/phaino/), [code](https://github.com/kubernetes-sigs/prow/tree/main/prow/cmd/phaino)) runs an approximation of a ProwJob on your local workstation
```

Comment:
The source code of phaino does not exist in either 
- https://github.com/kubernetes/test-infra/tree/master/prow/cmd/phaino or 
- https://github.com/kubernetes-sigs/prow/tree/main/prow/cmd/phaino

Reply:
`phaino` was removed in this PR: https://github.com/kubernetes/test-infra/pull/32312
We can delete all references to it in the docs.